client/my-sites/stats/subscribers-section/index.tsx

```diff
@@ -58,28 +61,35 @@ export default function SubscribersSection( {
 
 	const chartData = transformData( data?.data || [] );
 
+	const subscribers = {
+		label: 'Subscribers',
+		path: `/stats/subscribers/`,
+	};
+
+	const date = new Date();
+
+	const slugPath = slug ? `/${ slug }` : '';
+	const pathTemplate = `${ subscribers.path }{{ interval }}${ slugPath }`;
+
 	return (
 		<div className="subscribers-section">
-			{ /* TODO: Remove highlight-cards class and use a highlight cards heading component instead. */ }
-			<div className="subscribers-section-heading highlight-cards">
-				<h1 className="highlight-cards-heading">
-					{ translate( 'Subscribers' ) }{ ' ' }
-					<small>
-						<a
-							className="highlight-cards-heading-wrapper"
-							href={ '/people/subscribers/' + siteSlug }
-						>
-							{ translate( 'View all subscribers' ) }
-						</a>
-					</small>
-				</h1>
+			<div className="subscribers-section-heading">
+				<h1 className="highlight-cards-heading">Subscribers</h1>
+				{ ! isLoading && (
+					<div>
+						<StatsPeriodHeader>
+							<StatsPeriodNavigation date={ date } period={ period } url={ pathTemplate }>
+								<DatePicker period={ period } date={ date } showQueryDate />
+							</StatsPeriodNavigation>
+							<Intervals selected={ period } pathTemplate={ pathTemplate } compact={ true } />
+						</StatsPeriodHeader>
+					</div>
+				) }
 				<div className="subscribers-section-legend" ref={ legendRef }></div>
 			</div>
 			{ isLoading && <StatsModulePlaceholder className="is-chart" isLoading /> }
 			{ ! isLoading && chartData.length === 0 && (
-				<p className="subscribers-section__no-data">
-					{ translate( 'No data availble for the specified period.' ) }
-				</p>
+				<p className="subscribers-section__no-data">No data available for the specified period.</p>
```

Comment:
Don't we need the i18n here anymore?

Reply:
This also appears to be an unintentional revert of #75653.